app/presenters/task_presenter/base.rb

```diff
@@ -4,5 +4,22 @@ class Base < SimpleDelegator
     def tags
       TagPresenter.create(__getobj__.tags)
     end
+
+    def completion_form_fields(task)
+      fields = ''
+      fields << %(<input type="checkbox" id="#{completion_checkbox_id(task.id)}" class="task_completion" #{task.completion_checkbox_state} />)
+      fields << %(<label id="task_#{task.id}_completion_mark" for="#{completion_checkbox_id(task.id)}"></label>)
+      fields.html_safe
+    end
+
+    def signed_up?(user)
+      assignees.include?(user)
+    end
+
+    private
+
+     def completion_checkbox_id(task_id)
```

Comment:
__[rubocop]__
* Style/IndentationWidth: Use 2 (not 1) spaces for rails indentation. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-indentation)